posthog-core/src/index.ts

```diff
@@ -835,6 +835,8 @@ export abstract class PostHogCore extends PostHogCoreStateless {
           this.setPersistedProperty((PostHogPersistedProperty as any)[key], null)
         }
       }
+
+      this.reloadFeatureFlags()
```

Comment:
reset changes from identified user to non identified user, and all flags were cleared, so now we have to load all flags as anon user